src/Utilities/SetHistory.js

```diff
@@ -0,0 +1,17 @@
+import queryString from 'query-string';
+
+export function setHistory(history, systemIds, clearState) {
+    if (clearState === true) {
```

Comment:
I am not seeing the need for clearState. This function is not additive so it will always reset the set of systemIds on the history. I would consider performing this based on an empty set of systemIds or create a clearHistory function. This might be useful to consider in light of wanting to also store filters on the URL.